GIT_CONFLICTS.md

```diff
@@ -0,0 +1,77 @@
+# Wanderlust Project
+
+## Overview
+Welcome to the Wanderlust project! This is a collaborative effort to build an amazing application. We appreciate your contributions and aim to make the process as smooth as possible.
+
+## Common QnA for Git and Open Source Contributions
+

```

Comment:
Let's have this as heading,
 "Common QnA"

Reply:
done